tests/integ/test_pandas_to_df.py

```diff
@@ -549,3 +564,349 @@ def test_write_to_different_schema(session, local_testing_mode):
     finally:
         if not local_testing_mode:
             Utils.drop_schema(session, test_schema_name)
+
+
+@pytest.mark.localtest
```

Comment:
This marker is no longer needed.

Reply:
I have a PR that removes these: https://github.com/snowflakedb/snowpark-python/pull/1562